putio.py

```diff
@@ -9,6 +9,15 @@
 import requests
 import iso8601
 
+KB = 1024
+MB = 1024 * KB
+
+# Downloads are broken up into ranges of this size
+RANGE_SIZE = 10 * MB
```

Comment:
Ok, I think easiest and probably the best thing to do is to get rid of the repeated ranges and specify one big range for the leftover stuff as you've said above. The chunk size can be the tweakable bit. The only other advantage I can see with ranges is to use parallel calls to the server, but we're not there yet. :)